src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/Lambda.scala

```diff
@@ -74,23 +118,45 @@ class Lambda extends RequestHandler[java.util.Map[String, Object], APIGatewayPro
     inputParameters.get("consignmentId").toString
   }
 
-  private def validateMetadata(draftMetadata: DraftMetadata, schema: Set[JsonSchemaDefinition]): IO[Map[String, Seq[ValidationError]]] = {
+  private def validateMetadata(draftMetadata: DraftMetadata, csvData: List[FileRow], schema: Set[JsonSchemaDefinition]): Either[Unit, ErrorFileData] = {
+    val validationErrors = MetadataValidationJsonSchema
+      .validate(schema, csvData)
+      .collect {
+        case result if result._2.nonEmpty =>
+          val errors = result._2.map(p =>
+            Error(
+              p.validationProcess.toString,
+              convertToAlternateKey(defaultAlternateKeyType, p.property),
```

Comment:
if there is no alternate key the value is "" 
probably better to just use the property